certified-connectors/Revelation helpdesk/README.md

```diff
@@ -0,0 +1,52 @@
+﻿
+## Revelation Helpdesk Connector
+The Revelation helpdesk connector allows you to create and update items in your helpdesk such as tickets, clients, users and assets by connecting to the Revelation helpdesk API using OAuth authentication. You can also take advantage of the extensive list of triggers allowing you to integrate your business processes based on events that occur in Revelation helpdesk.
+
+
```

Comment:
@MVliegenthart Please refer the template provided readme.md file and add required sections.

Reply:
@sriyen-msft I have added the additional required sections.